use-cases.md

```diff
@@ -5,7 +5,7 @@ sectionid: use-cases
 canonical: 'https://druid.apache.org/use-cases'
 ---
 
-## Real-time analytics and intelligence
+## Power data applications
```

Comment:
This headline is too specific for the content below (many of the use cases are data applications, but not all of them are).

Suggest rewording it to something like:

> Power real-time analytics, data applications, and more